docs/docs/testing/e2e/setup.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Setup
+tags:
+  - testing
+  - e2e
+  - playwright
+  - infrastructure
+---
+
+## Overview
+
+This document explains at a high level how we create our app instance, run the e2e tests and what technology we're using for e2e tests. As well as a small section on writing tests.
+
+## Get Started
+
+Because we require a "fresh" instance to assert our e2e tests against this is included in the testing script so all you need to run is:
+
+```shell
+yarn test:e2e
+```
```

Comment:
````suggestion
To create the initial test apps and install the Playwright browser dependencies, run:

```shell
yarn test:e2e:setup
npx playwright install
```

After the initial setup, the scripts to reset the test app to the initial state are run automatically by the test command:

```shell
yarn test:e2e
```
````

Just a note on some things I had to figure out when `yarn test:e2e` didn't just work.

Reply:
Yes thanks :) I will update these guides too - I think the whole branch became a bit outdated over time :)